.github/workflows/testing.yml

```diff
@@ -26,42 +31,70 @@ jobs:
       - name: Check resolving dependencies did not change the lock file
         run: |
           echo "Checking if locking dependencies has created any diff in the lock file..."
-          if 
-            git diff --exit-code -- poetry.lock
-          then
-            echo "No changes to poetry.lock detected"
-          else
+          set +e
+          git diff --exit-code -- poetry.lock > /dev/null
```

Comment:
I intentionally had the git diff going to stdout so that someone reviewing a failing check can see the diff, can you restore the old code here.